libnavigation-core/src/main/java/com/mapbox/navigation/core/fasterroute/FasterRouteObserver.kt

```diff
@@ -16,11 +16,11 @@ interface FasterRouteObserver {
     /**
      * Invoked whenever a faster route was inspected.
      *
-     * @param currentRoute reference to the current route.
-     * @param alternativeRoute reference to the alternative checked.
-     * @param isAlternativeFaster true if the alternativeRoute is faster, false otherwise
+     * @param currentRoute the current route.
+     * @param alternatives the routes where alternatives[0] may be faster
+     * @param isAlternativeFaster true if the alternatives[0] is faster, false otherwise
      */
-    fun onFasterRoute(currentRoute: DirectionsRoute, alternativeRoute: DirectionsRoute, isAlternativeFaster: Boolean)
+    fun onFasterRoute(currentRoute: DirectionsRoute, alternatives: List<DirectionsRoute>, isAlternativeFaster: Boolean)
```

Comment:
For documentation purposes noting that this breaks SemVer cc @langsmith

Reply:
Thanks @Guardiola31337